File: Java/프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.java
import java.util.Stack;

class Solution {
    public int solution(String s) {
        Stack<Integer> stack = new Stack<>();
        String[] temp = s.split("\\s+");
        int answer = 0;
        int total = 0;
        for(String t : temp){
            if(t.equals("Z")){
                int it = stack.pop();
                total -= it;
            }else{
                int it2 = Integer.parseInt(t);
                total += it2;
                stack.push(it2);
            }
        }
        answer = total;
        return answer;
    }
}

File: Python3/프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    arr = []
    arr = map(str, s.split(' '))
    
    stack = []
    for x in arr:
        if x == 'Z':
            stack.pop()
        else:
            stack.append(int(x))
    answer = sum(stack)
    return answer